hibernate-core/src/main/java/org/hibernate/type/descriptor/sql/ClobTypeDescriptor.java

```diff
@@ -126,7 +126,7 @@ protected void doBind(PreparedStatement st, X value, int index, WrapperOptions o
 							CharacterStream.class,
 							options
 					);
-					st.setCharacterStream( index, characterStream.asReader(), characterStream.getLength() );
+					st.setCharacterStream( index, characterStream.asReader() );
```

Comment:
I think this change is unnecessary

Reply:
thanks -- went back to the original version